i18n/de-DE.json

```diff
@@ -1107,6 +1107,9 @@
 				"available": "Available Users",
 				"selected": "Selected Members",
 				"okButton": "Confirm members"
+			},
+			"toast": {
+				"emailError": "One or more Email Recipients are invalid."
```

Comment:
Add a reminder to this text, to indicate that the e-mails should be comma-separated.